internal/scm/bitbucketserver/bitbucket_server_test.go

```diff
@@ -0,0 +1,99 @@
+package bitbucketserver
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	bitbucketv1 "github.com/gfleury/go-bitbucket-v1"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBitbucketServer_GetRepositories_SkipEmptyRepo(t *testing.T) {
+	// Mock Bitbucket Server
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		log.Printf("Mock server received request: %s %s", r.Method, r.URL.Path)
+		if strings.HasSuffix(r.URL.Path, "/rest/api/1.0/projects/PROJ/repos") { // GetRepositoriesWithOptions
```

Comment:
Is the `strings.HasSuffix`es necessary here? Seems that the path would be exact matches